core/server/models/post.js

```diff
@@ -237,7 +237,7 @@ Post = ghostBookshelf.Model.extend({
             this.set('html', legacyConverter.render(_.toString(this.get('markdown'))));
         }
 
-        if (this.hasChanged('html')) {
+        if (this.hasChanged('html') || !this.get('plaintext')) {
```

Comment:
I'm not sure why the [test for this](https://github.com/TryGhost/Ghost/blob/master/core/test/integration/model/model_posts_spec.js#L433-L448) wasn't failing before or why it started to with this PR as the test doesn't change the HTML